src/index.html

```diff
@@ -12,9 +13,184 @@
     <meta name="theme-color" content="#ffffff">
 
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <style>
+        noscript {
```

Comment:
@zainzafar90 can we put this style in styles.scss file?

Reply:
We can load into a separate file but let me check if we can load into styles.scss, because some styles get inject using JS so I took it out intentionally.

Reply:
By the way I used this inline style because it wasn't loading styles from theme at all. So what I can do check if it's possible, if not I'll store it in separate css file

Reply:
@zainzafar90 keep this file inside assets and include it directly inside html